util/i2csvg/examples/SMBusCommands.txt

```diff
@@ -0,0 +1,102 @@
+T 6.5.1 Quick Command
+SPA2
+
+T 6.5.2 Send Byte
+SA0
+P'Data'
+
+T 6.5.3 Receive Byte
+SA1
+RP1
```

Comment:
Question: Do the flags have to be in any particular order?

Reply:
On input flags can be in any order (and dots are ignored so the pretty-print output could be used as input).

Should add a comment in the text.

Reply:
comment added